src/main/java/lotto/model/LottoPrice.java

```diff
@@ -0,0 +1,37 @@
+package lotto.model;
+
+import lotto.util.ErrorMessage;
+import lotto.util.exception.CustomException;
+
+public class LottoPrice {
+    private final int price;
+
+    public LottoPrice(String price) {
+        this.price = validate(validateIsIntPrice(price));
+    }
+
+    private int validateIsIntPrice(String price) {
+        int priceNumber = 0;
+        try {
+            priceNumber = Integer.parseInt(price);
+        } catch (NumberFormatException e) {
+            throw new CustomException(ErrorMessage.LOTTO_PRICE_MUST_BE_INTEGER);
+        }
+        return priceNumber;
```

Comment:
```suggestion
        try {
            return Integer.parseInt(price);
        } catch (NumberFormatException e) {
            throw new CustomException(ErrorMessage.LOTTO_PRICE_MUST_BE_INTEGER);
        }
```
바로 리턴하는 방식으로 구현하면 조금 더 짧아지고 불필요한 선언도 없어질 것 같습니다